<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-57L41J9" failures="3" tests="6" name="Edso" time="0.33" errors="0" timestamp="2023-02-27T15:59:32 IST">
  <testcase classname="demo.Reporting.ReportingExample" name="Login1" time="0.026"/>
  <testcase classname="demo.Reporting.ReportingExample" name="Login2" time="0.042">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.Reporting.ReportingExample.Login2(ReportingExample.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Login2 -->
  <testcase classname="demo.Reporting.ReportingExample" name="Login3" time="0.0"/>
  <testcase classname="demo.Reporting.ReportingExample" name="Login4" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.Reporting.ReportingExample.Login4(ReportingExample.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Login4 -->
  <testcase classname="demo.Reporting.ReportingExample" name="Login5" time="0.001"/>
  <testcase classname="demo.Reporting.ReportingExample" name="Login6" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.Reporting.ReportingExample.Login6(ReportingExample.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Login6 -->
</testsuite> <!-- Edso -->
